docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -124,10 +129,13 @@ When updating extension configuration:
 
 == Lint documentation changes with Vale
 
-Our builds use https://vale.sh[Vale] to check grammar, style, and word usage in the English versions of our documents.
-The configuration for vale is in the `docs/.vale` directory.
+Our builds use https://vale.sh[Vale] to check grammar, style, and word usage in the English versions of our documents. We created our own Vale style ruleset to ensure that content aligns with the preferred Quarkus style guidelines.
 
-NOTE: The Vale configuration file is `docs/.vale/vale.ini`.
+[NOTE]
+====
+* The Quarkus configuration for vale is `vale.ini`, which is in the `docs/.vale` directory.
```

Comment:
watch delivery of #29913 ... conflict here

Reply:
@ebullient - Thanks for flagging. I'm now :eyes: subscribed to PR [29913](https://github.com/quarkusio/quarkus/pull/29913).